### ISSUE TITLE
Gapless playing of the songs 

### ISSUE THREAD

**Issue by LjupcoStojkoski** [Mastermindzh/tidal-hifi#379]
I have installed the latest version (5.10.0) of opensuse tumbleweed, everything works fine, except gapless play of the songs in albums where there should be no pause between songs. My account is Premium TIDAL with Hi Res. Is there any solution ? Thx

**Comment by Mastermindzh** [comment]
@LjupcoStojkoski , can you give me the name of the album? I can check it then 😄.

**Comment by LjupcoStojkoski** [comment]
sure :) , I will just pick one of the many ;)
https://tidal.com/browse/album/7909805?u
The Wall - Pink Floyd (example, between the songs No. 2-3, 4-5 ..... )

**Comment by Mastermindzh** [comment]
Hey,

It seems I don't usually have gapless albums since I've never noticed it before... 
But it's definitely there, I also found another [album](https://listen.tidal.com/album/186865973) (from songs I do sometimes listen) and it has the same effect jarring effect...

Unfortunately it also shows up on two official TIDAL sources:

- https://listen.tidal.com
- Android app

All using the same account (with all options enabled).

So I can't help but conclude that this is (unfortunately) a TIDAL upstream problem.
If I google around a little bit I see a LOT of people having issues with it as well :/ shame.

**Comment by sinewavess** [comment]
> So I can't help but conclude that this is (unfortunately) a TIDAL upstream problem. If I google around a little bit I see a LOT of people having issues with it as well :/ shame.

I tried the windows client and gapless works perfectly fine on there

**Comment by LjupcoStojkoski** [comment]
> > So I can't help but conclude that this is (unfortunately) a TIDAL upstream problem. If I google around a little bit I see a LOT of people having issues with it as well :/ shame.
> 
> I tried the windows client and gapless works perfectly fine on there

Unfortunately, I cannot try on Windows, but it is strange how is that possible if the upstream  is the same (or not) :(

**Comment by DeN-AlB** [comment]
It seems to be an issue with Pipewire, like the creator of High-Tide states in the settings of his app.

![Image](https://github.com/user-attachments/assets/dea77abd-f989-4b64-83b2-2972be6bd7d9)

He added support for Gapless Playback in version 1.0.0.

https://github.com/Nokse22/high-tide/releases/tag/v1.0.0

Maybe this could be added into Tidal Hifi?

**Comment by Mastermindzh** [comment]
@DeN-AlB,

Tidal-Hifi doesn't change anything in the playback from TIDAL directly.
Once they start supporting gapless playback on the web, so will Tidal-Hifi :)

**Comment by DeN-AlB** [comment]
Good to know. Thanks, @Mastermindzh !

Is there any place where you can vote for such a feature request on Tidal? 🤔

**Comment by Mastermindzh** [comment]
@DeN-AlB , Yes sir, on their website: https://support.tidal.com/hc/en-us/articles/201937771-Feature-and-Content-Requests

Unfortunately there isn't an issue tracker or something else like it :/

**Comment by DeN-AlB** [comment]
Thanks a lot. I’m new at Tidal and want to try it out to move away from Spotify. I‘ll look into their support system. 👍
